src/rattler_build_conda_compat/jinja.py

```diff
@@ -27,7 +27,28 @@ def jinja_env() -> jinja2.Environment:
     """
     Create a `rattler-build` specific Jinja2 environment with modified syntax.
     """
-    return jinja2.Environment(
+
+    def stub_compatible_pin(*args, **kwargs) -> str:  # noqa: ARG001, ANN003, ANN002
+        return f"compatible_pin {args[0]}"
+
+    def stub_subpackage_pin(*args, **kwargs) -> str:  # noqa: ARG001, ANN003, ANN002
+        return f"subpackage_pin {args[0]}"
+
+    def version_to_build_string(some_string: str) -> str:
+        """Converts some version by removing the . character and returning only the first two elements of the version)"""
+        # We first split the string by whitespace and take the first part
+        split = some_string.split()[0] if some_string.split() else some_string
+        # We then split the string by . and take the first two parts
+        parts = split.split(".")
+        major = parts[0] if len(parts) > 0 else ""
+        minor = parts[1] if len(parts) > 1 else ""
+        return f"{major}{minor}"
```

Comment:
I can imagine that it would be useful to add a test for this implementation too.

Reply:
added!